pocs/utils/images/__init__.py

```diff
@@ -83,27 +94,28 @@ def make_pretty_image(fname, title=None, timeout=15, link_latest=False, **kwargs
     Returns:
         str -- Filename of image that was created.
     """
-    assert os.path.exists(fname),\
+    if not os.path.exists(fname):
         warn("File doesn't exist, can't make pretty: {}".format(fname))
-
-    if fname.endswith('.cr2'):
+        return None
+    elif fname.endswith('.cr2'):
         pretty_path = _make_pretty_from_cr2(fname, title=title, timeout=timeout, **kwargs)
     elif fname.endswith('.fits'):
         pretty_path = _make_pretty_from_fits(fname, title=title, **kwargs)
     else:
         warn("File must be a Canon CR2 or FITS file.")
         return None
 
+    if not link_latest or not os.path.exists(pretty_path):
+        return pretty_path
+
     # Symlink latest.jpg to the image; first remove the symlink if it already exists.
-    if link_latest and os.path.exists(pretty_path):
-        latest_path = os.path.join(
-            os.getenv('PANDIR'),
-            'images',
-            'latest.jpg'
-        )
+    images_dir = make_images_dir()
```

Comment:
Having this called from `make_pretty_image` seems a bit odd given it's name. Might make more sense to have a `get_images_dir` that just silently handles making the directory.